packages/core/rpc/BaseRpcDriver.ts

```diff
@@ -62,6 +62,9 @@ export default abstract class BaseRpcDriver {
         .map(t => this.filterArgs(t, pluginManager, stateGroupName))
     }
     if (typeof thing === 'object' && thing !== null) {
+      // null is special of course
```

Comment:
there is a check for thing !== null above that i added recently :)

Reply:
ah yep i'll de-dup that

Reply:
note that I also had it console.warn if null does arise because I was confused about when it happens, but I think it does happen naturally in some trackList.json's in jb1, they just have null in trackType sometimes